docs/orchestrate/private-registry.md

```diff
@@ -0,0 +1,69 @@
+---
+id: private-registry
```

Comment:
I'd rename it to `container-registry`. It's not about private registries, it's about auth-protected images in a public registry.